### PR TITLE
MVC Layout + Login Work

### PR DESCRIPTION
**Pull request by s3494110**
@s3364859 @6nop @s3364942 @s3482247 

In this PR:
- Website cosmetic stuff. 
- Login system for the admin portal.

Neither are 100% finished, but a PR is probably due for the site.

### CODE REVIEW

**Comment by 6nop** [comment]
@s3494110 Angus,

Dapper doesn't seem to like the "Area" geography attribute of the Location].LocalGovernmentAreaState table.

I've tried a bunch of fixes, but the simplest was to alter the Stored Proc - spGetLocalGovernmentAreaState returns the Area value as text.

```
ALTER PROCEDURE [Location].[spGetLocalGovernmentAreaState]
AS
SELECT  
	ID,
	StateID,
	StateName,
	AbbreviatedName,
	Name,
	Area.STAsText(),
	IsDeleted,
	IsVisible,
	DateCreatedUtc,
	DateUpdatedUtc
FROM
	[Location].LocalGovernmentAreaState
ORDER BY
	StateName,
	Name

GO
```

This will need to be added to the migration scripts as **Location.spGetLocalGovernmentAreaState2.sql**, so it occurs after the create script.

@s3364859 Carl, would you prefer to update Location.spGetLocalGovernmentAreaState.sql and rerun the whole script, or create a Location.spGetLocalGovernmentAreaState2.sql and support incremental change?

File: CPT331.Web/Controllers/AdminController.cs [hunk 1]
```diff
@@ -15,10 +15,15 @@
...
 {
     public class AdminController : Controller
-    {		
-		[AcceptVerbs(HttpVerbs.Get)]
+    {
+        [AcceptVerbs(HttpVerbs.Get)]
         public ActionResult Home()
         {
+            // User isn't logged in. 
+            if (Session["user"] == null)
```
Comment (s3364859) [inline CPT331.Web/Controllers/AdminController.cs]: We should consider moving this to an action attribute.

**Comment by s3364859** [comment]
Overall I'm fine with these changes.

@6nop - I would just rewrite the sproc rather than create the migration, I have no issues rebuilding the database here and popping it up there if required.

@s3494110 - are we sure that the latest code has been merged into this PR?

**Comment by 6nop** [comment]
Local government issue has been resolved and we'll look at the improved log-in using a database and attributes in the next PR.

All good for now 👍

**Pull request merged**